### PR TITLE
Add v5.0.0 entry

### PR DESCRIPTION
**Pull request by kenhys**
TODO: add missing changelog entry

### CODE REVIEW

**Comment by kenhys** [comment]
To pick up already merged notable PR, write up a summary here.

**Comment by daipom** [comment]
Thanks for this!!
I'll add any changes I notice here as well!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,601 +1,181 @@
...
+* Amazon Linux 2023 has been supported. (#459)
+* Introduced new package signing key. The new key will be used in the future
+  release. we still use using old signing key for a while. (#507)
+
+Not only changing package name, but also there are some notable changes.
+Basically, for `td-agent` v4 users, it aims to keep compatibility as far as possible
+by executing the migration process with copying old files or providing
+symbolic links for it. For newly installed `fluent-package`, no such
+compatibility layer will not provided. (It means that fresh install should not provides
+such a redundant backward compatible symbolic links and so on)
```
Comment (daipom) [inline CHANGELOG.md:22]: > For newly installed `fluent-package`, no such ...

I think this description might be unnecessary.

**Review by daipom — APPROVED** [review]
I haven't confirmed the components, but the other parts look good to me.
I think we can improve the content about migration in future PRs.

File: CHANGELOG.md (hunk 1 above)
Comment (kenhys) [reply CHANGELOG.md]: It may be too negative wording.
Thanks, fixed it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,601 +1,179 @@
...
+
+* msi: stop customizing icons for file browser (#469)
+* msi: update resources for `fluent-package` (#470)
+* msi: changed the default path of buffer/failed_records (#527)
+
+#### For macOS user:
+
+* dmg: renamed to `fluent-package` (#474,#478,#479,#480,#481,#482,#483)
+  * Note that the .dmg package support will be dropped in the future
+  release. We plans to migrate for `homebrew` ecosystem.
```
Comment (ashie) [inline CHANGELOG.md:113]: We have currently no plan to release dmg version of fluent-package v5 and not tested enough, so we should notify it at first.

Comment (kenhys) [reply CHANGELOG.md:116]: Thanks, I've added it.

**Pull request merged**